flocker/provision/_effect.py

```diff
@@ -80,9 +85,48 @@ def reducer(next_effect, effect):
     return reduce(reducer, reversed(effects), results)
 
 
+class HTTPGet(PClass):
+    """
+    Intent for HTTP GET requests.
+
+    :ivar bytes url: The URL to make a GET request to.
+    """
+    url = field(type=bytes)
+
+
+def http_get(url):
+    """
+    Wrapper to create an :class:`HTTPGet` Effect.
+
+    :param bytes url: The url to make a GET request to.
+
+    :returns: The ``Effect`` of making a GET request to ``url``.
+    """
+    return Effect(HTTPGet(url=url))
+
+
+@deferred_performer
+def treq_get(dispatcher, intent):
+    """
+    Performer to execute an HTTP GET.
+
+    :param dispatcher: The dispatcher used to dispatch this performance.
+    :param HTTPGet intent: The intent to be performed.
```

Comment:
+1 on it not adding any value... unless it lets us use an automated pylint-style checker to verify all new parameters are documented.